src/main/resources/db/migration/V17__create_invitation_table.sql

```diff
@@ -0,0 +1,15 @@
+-- V2__create_invitation_table.sql
+CREATE TABLE invitation_table (
+    id VARCHAR(255) PRIMARY KEY,
+    token VARCHAR(255) NOT NULL UNIQUE,
+    user_email VARCHAR(255) NOT NULL,
+    status VARCHAR(20) NOT NULL,
+    expires_at TIMESTAMP NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
check the repo and update the version no accordingly.

Reply:
Done